jdaviz/configs/imviz/plugins/catalogs/catalogs.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+   <j-tray-plugin
+     description='Queries an area encompassed by the viewer using the SDSS and marks all the objects found within the area in the given catalog.'
+    :link="'https://jdaviz.readthedocs.io/en/latest/dev/ui_plugin_components.html'"
+    :popout_button="popout_button">
+
+     <v-row>
+       <v-select
+         :items="viewer_items"
+         v-model="selected_viewer"
+         label="Viewer"
+         hint="Select a viewer to search."
+         persistent-hint
+       ></v-select>
+     </v-row>
+
+     <v-row justify="end">
+       <v-btn color="primary" text @click="do_catalogs">SDSS Search</v-btn>
```

Comment:
Not sure if this is good enough to inform the user of the service being used. Otherwise, should I change the structure of this up or add more to it?